pilot/pkg/config/kube/crdclient/cache_handler.go

```diff
@@ -36,11 +43,15 @@ import (
 // cacheHandler abstracts the logic of an informer with a set of handlers. Handlers can be added at runtime
 // and will be invoked on each informer event.
 type cacheHandler struct {
-	client   *Client
-	informer cache.SharedIndexInformer
-	handlers []func(config.Config, config.Config, model.Event)
-	schema   collection.Schema
-	lister   func(namespace string) cache.GenericNamespaceLister
+	client              *Client
+	informer            cache.SharedIndexInformer
+	handlers            []func(config.Config, config.Config, model.Event)
+	schema              collection.Schema
+	lister              func(namespace string) cache.GenericNamespaceLister
+	currentObjMap       sync.Map
```

Comment:
why need this

Reply:
I am a little concerned this will increase the memory cost in big degree. Maybe just for resources(serviceentry) that need to compare